packages/data/addon/models/bard-dimension.js

```diff
@@ -6,7 +6,11 @@
  */
 import Ember from 'ember';
 
-export default Ember.Object.extend().reopenClass({
+export default Ember.Object.extend({
+  isEqual: function(other) {
```

Comment:
now we can use the arrow function right?

Reply:
Nah, `this` isn't available here: gave it a shot and it broke everything.